client/src/components/menu.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+import Home from './home.js'
+import About from './about.js'
+import Categories from './categories.js'
+ 
+import { BrowserRouter as Router, Route, Link } from "react-router-dom";
+
+
+class Menu extends Component {
+    render() {
+        return (
+            <Router>
```

Comment:
The Menu should just be the stuff inside `<nav>`  - the rest of the Router should stay at App.js

Reply:
also functional component - not class